frontend/src/text_resources.js

```diff
@@ -725,8 +727,10 @@ let LOCALIZE = new LocalizedStrings({
       disabled: "Désactivé",
       backToTop: "Haut de la page",
       notepad: {
-        title: "Bloc-notes",
-        placeholder: "Mettez vos notes ici..."
+        title: "BLOC-NOTES",
```

Comment:
In this document, strings can be regular casing, and then in JS you can use the .toUpperCase() function.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/150/commits/79a06e27fcf2990a4ce2e1e1404769383a382bec